Develop3

R4M5EY opened the pull request.

R4M5EY committed:
add tests

R4M5EY committed:
add allure report (and loginTest)

R4M5EY committed:
add allure results

alesyaisachenko commented on .idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
папку idea нужно добавлять в гитигнор

alesyaisachenko commented on src/main/java/pages/ForgotPasswordPage.java:
```diff
@@ -0,0 +1,34 @@
+
+public class ForgotPasswordPage {
+    WebDriver driver;
+    public final String url = "https://stellarburgers.nomoreparties.site/forgot-password";
```
лучше public static final String URL -- в Java так принято именовать константы

коммент относится ко всему коду

alesyaisachenko commented on src/test/java/ConstructorTest.java:
```diff
@@ -0,0 +1,75 @@
+    }
+
+    @Parameterized.Parameters
+    public static Object[][] getBrowser() {
```
через параметризацию тесты не запускают в разных браузерах.

запуск тестов в разных браузерах настраивают иначе. Тем более не все тесты прогоняют на всех поддерживаемых браузерах.

Параметризацию для запуска тестов нужно обязательно убрать во всем коде.

alesyaisachenko commented on src/test/java/ConstructorTest.java:
```diff
@@ -0,0 +1,75 @@
+    }
+
+    @Before
+    public void setup() {
```
это общий код для всех тестов. Логично его вынести в одно место (BaseTest), а затем его использовать во всех тестах (наследоваться от BaseTest)

R4M5EY committed:
delete parameterization for browsers
add new allure results
add idea to gitignore and other minor changes from the comments

R4M5EY committed:
removed unused imports
